test/util/zip.js

```diff
@@ -12,6 +12,44 @@ const streamTest = require('streamtest').v2;
 //      {filename}: "contents",
 //      …
 // }
+const processZipFile = (zipfile, callback) => {
+  const result = { filenames: [] };
+  const entries = [];
+  let completed = 0;
+
+  zipfile.on('entry', (entry) => entries.push(entry));
+  zipfile.on('end', (err) => {
+    if (err) return callback(err);
```

Comment:
More a note-to-self, previously `zipfile.on('end')` didn't check for an error passed to it.

Reply:
Maybe that was the entire problem :thinking: I'll check

Reply:
That wasn't the problem.  Maybe that `err` argument can be removed from the new code.

Reply:
Either way, probably doesn't hurt to keep it.